pkg/config/evp_proxy.go

```diff
@@ -0,0 +1,15 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package config
+
+// setupEvpProxy initializes the configuration values of the evp_proxy.
+func setupEvpProxy(cfg Config) {
+	cfg.BindEnv("evp_proxy_config.enabled")
```

Comment:
Any reason you wanted a separate file and function? I'd just put them in `pkg/config/config.go`

Reply:
I saw that appsec got a separate file so I did the same, but I agree it can be on the same one. Let me change it.